src/components/calcite-action/calcite-action.scss

```diff
@@ -66,26 +66,6 @@
   width: 100%;
 }
 
-.button--text-interactive {
```

Comment:
Getting rid of this does clean things up, but we are losing the animation. Is there a way to keep the animation?

Reply:
Yes, ill keep it.

Reply:
Cool.